pkg/activator/handler/handler.go

```diff
@@ -20,33 +20,48 @@ import (
 	"net/url"
 	"strconv"
 	"time"
-
 	"github.com/knative/serving/pkg/activator"
 	"github.com/knative/serving/pkg/activator/util"
 	pkghttp "github.com/knative/serving/pkg/http"
 	"go.uber.org/zap"
 )
 
+const overloadMessage = "throttler overload"
+
 // ActivationHandler will wait for an active endpoint for a revision
 // to be available before proxing the request
 type ActivationHandler struct {
 	Activator activator.Activator
 	Logger    *zap.SugaredLogger
 	Transport http.RoundTripper
 	Reporter  activator.StatsReporter
+	Throttler *activator.Throttler
 }
 
 func (a *ActivationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	namespace := pkghttp.LastHeaderValue(r.Header, activator.RevisionHeaderNamespace)
 	name := pkghttp.LastHeaderValue(r.Header, activator.RevisionHeaderName)
 
 	start := time.Now()
+	revID := activator.RevisionID{namespace, name}
+	a.Logger.Debug(revID)
+	a.Throttler.AddIfAbsent(revID)
```

Comment:
Return value is not checked here. If it doesn't matter why would we return anything?